### DISCUSSION TITLE
Call for Votes : Should we accept Celebian as Sponsor?

### DISCUSSION THREAD

**Discussion by benjagm** [json-schema-org/community#580] [General]
### Context
Past week I was excited about a new JSON Schema sponsor request by [Celebian](https://celebian.com/) because that happened after the work done to improve the  [Sponsorship packages](https://json-schema.org/overview/sponsors) and to refresh our [OpenCollective profile](https://opencollective.com/json-schema).  The first thing we did was to check the acceptance policy and in this process [some doubts have been raised](https://github.com/json-schema-org/community/issues/570) by some community members.

### Research
We are not experts in the Tik-Tok business in which the prospect sponsor operates but buying/selling likes and followers has been pointed as potentially unethical by some community members. 

Arguments received in favour of accepting the Sponsor:
- In their Github repo https://github.com/celebiancom the show that they are actually supporting other relevant OpenSource projects like: Django, Curl and Nuxt.
- Trustpilot reviews.

Arguments against accepting the Sponsor:
 - buying/selling likes and followers could be perceived as potentially unethical.
 - [Trustpilot reviews not validated](https://github.com/json-schema-org/community/issues/570#issuecomment-1892849383) If they can generate views and followers on Tik-Tok they can generate review on TrustPilot.
 
 ### Next steps
The intend of this issue is:

* **Step 1**: To check if we have more arguments in favour or against accepting the sponsor. 
Action: Please add the arguments as comments on this issue.

* **Step 2**:  Ask the TSC for vote about finally accepting the sponsor or not with following the [TSC voting process](https://github.com/json-schema-org/community/blob/main/GOVERNANCE.md). Additionally we'll appreciate opinions and comments coming from anyone in the Community. Everyone is welcome!!
Action: Use this issue to follow the voting process using 👍 or 👎 reactions.

Please, join the discussion and help us to take the best decision for JSON Schema! 🙏

**Comment by notEthan** [comment]
I'm curious how the acceptance criteria were selected? I see they are taken from the policy on OpenCollective (per 
https://github.com/json-schema-org/community/pull/516#issuecomment-1822452726), and can see many of them being vaguely objectionable in principle, though not really in a way particular to the JSON Schema org. Others - in particular financial and pharmaceutical - only seem objectionable in that the companies that provide them often do evil things, but I don't know that these are generally against any principle of the organization.

That is a broader question than this discussion asks and may be out of scope. In scope: However those criteria are picked, rejecting a company that trades in manufactured social media engagement seems in keeping with rejection of essay writing services, for selling a product that is basically dishonest.

**Reply by Relequestual** [reply]
> I'm curious how the acceptance criteria were selected? I see they are taken from the policy on OpenCollective

OpenCollective have a dropdown list of types of companies that you can choose to not accept sponsorship from. I guess those are the categories they see people wanting to reject most.
At some point, seeing that list, I selected a number of them (listed) that I thought would be unequivocal.

**Comment by gregsdennis** [comment]
I'm personally indifferent here.

For me it just takes recognition that social media _businesses_ (e.g. TikTok) are not in the game to benefit society.  As such, businesses like Celebian are just a cog in that machine.

Would we accept sponsorship from TikTok, Facebook, or Twitter?  I imagine that Celebian has to go in that same bucket.

Yes, the social media engagement they sell is manufactured, but they're honest about what they do, and from what I can tell, they do what they say.

**Comment by Julian** [comment]
I'd be a (weak) -1 -- for just the reasons mentioned in the original post -- it's ethically questionable as a practice.

**Comment by benjagm** [comment]
Quick note on the [Voting Process](https://github.com/json-schema-org/community/blob/main/GOVERNANCE.md):  I realized that I was not following the process this time. Next time we will:

1. Use an issue in the TSC repository instead of a discussion.
2. Use the `tsc-vote` label.
3. User reactions in the first comment to provide the vote.

My apologies for the confusion.

**Comment by karenetheridge** [comment]
I vote no, do not accept; we should avoid applications or organizations that contribute to an unhealthy internet (and I would count fake data and AI-generated content as unhealthy).

**Comment by jdesrosiers** [comment]
Although I'm generally a lot more accepting than our stated rules currently allow, I'm a :-1:  for this one. This product's purpose is to deceive social media users and manipulate the spread of information. I might be more forgiving if there was a guarantee that any effected post was required to be clearly labeled as having bought so many likes/views/whatever. If that were the case, it could be argued that it's a valid advertising/promotion market, but the hidden nature of this information manipulation makes this product potentially dangerous and I'd rather not have anything to do with that.

**Comment by benjagm** [comment]
Thanks everyone who provided its vote. We did reach out to Celebian to communicate our decision.

Additionally, this type of business will be added to the [rejection criteria](https://github.com/json-schema-org/community/blob/main/programs/sponsors/sponsors.md#acceptance-policy).

This conversation will be closed as we completed the voting process.
